lib/daftka/metadata/store.ex

```diff
@@ -220,7 +253,7 @@ defmodule Daftka.Metadata.Store do
       topic_key = Types.topic_value(topic)
       partition_index = Types.partition_value(partition)
 
-      Agent.get(__MODULE__, fn %State{topics: topics} ->
+      Agent.get(name_for_calls(), fn %State{topics: topics} ->
```

Comment:
add a # TODO here that we can consider using gproc for this purpose (topic-partition -> pid) instead